lib/azure/armrest/storage_account_service.rb

```diff
@@ -23,52 +25,50 @@ def initialize(_armrest_configuration, _options = {})
       #
       # Example:
       #
-      #   sam.get('portalvhdstjn1ty0dlc2dg')
-      #   sam.get('portalvhdstjn1ty0dlc2dg', 'Default-Storage-CentralUS')
+      #   sas.get('portalvhds1234', 'Default-Storage-CentralUS')
       #
       def get(account_name, group = armrest_configuration.resource_group)
-        set_default_subscription
-
         raise ArgumentError, "must specify resource group" unless group
 
-        @api_version = '2014-06-01'
-        url = build_url(armrest_configuration.subscription_id, group, account_name)
-
+        url = build_url(group, account_name)
         JSON.parse(rest_get(url))
       end
 
       # Returns a list of available storage accounts for the given subscription
       # for the provided +group+, or all resource groups if none is provided.
       #
-      def list(group = armrest_configuration.resource_group)
+      def list(group = nil)
         if group
-          @api_version = '2014-06-01'
-          url = build_url(armrest_configuration.subscription_id, group)
+          url = build_url(group)
           JSON.parse(rest_get(url))['value'].first
```

Comment:
Yes. I need to look at the other service classes because I don't think we actually need an array, either.